Cannot compile develop branch in android studio

Hello there,
I get the following error in android studio during compilation:

```
:compileBetagroupJava/home/strubul/WordPress-Android/src/org/xmlrpc/android/ApiHelper.java:412: error: cannot find symbol
    public static class SyncMediaLibraryTask extends HelperAsyncTask<List<?>, Void, Integer> {
                                                                     ^
  symbol:   class List
  location: class ApiHelper
```

I just have forked and cloned the branch and followed the instructions in the handbook:

```
strubul@ubuntu:~/WordPress-Android$ git status
 On branch develop
nothing to commit (working directory clean)
```

Am I doing something wrong? Any help would be appreciated.

--- a/src/org/xmlrpc/android/ApiHelper.java
+++ b/src/org/xmlrpc/android/ApiHelper.java
@@ -144,7 +144,7 @@ public class ApiHelper {
         public void onSuccess();
     }
 
-    public static class GetPostFormatsTask extends HelperAsyncTask<List<?>, Void, Object> {
+    public static class GetPostFormatsTask extends HelperAsyncTask<java.util.List<?>, Void, Object> {
         private Blog mBlog;
 
         @Override
@@ -409,7 +409,7 @@ public class ApiHelper {
         }
     }
     
-    public static class SyncMediaLibraryTask extends HelperAsyncTask<List<?>, Void, Integer> {
+    public static class SyncMediaLibraryTask extends HelperAsyncTask<java.util.List<?>, Void, Integer> {
         public interface Callback extends GenericErrorCallback {
             public void onSuccess(int results);
         }

